doc/classes/TabContainer.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TabContainer" inherits="Container" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A container that creates a tab for each child control, displaying only the active tab's control.
+		A container that creates a tab for each child control, displaying only the tab that corresponds to the currently selected tab's control.
 	</brief_description>
 	<description>
 		Arranges child controls into a tabbed view, creating a tab for each one. The active tab's corresponding control is made visible, while all other child controls are hidden. Ignores non-control children.
-		[b]Note:[/b] The drawing of the clickable tabs is handled by this node; [TabBar] is not needed.
+		[b]Note:[/b] The drawing and managment of the clickable tabs is handled by this node; [TabBar] is not needed.
+
+		The tabs are generated only by the first children of the TabContainer node. Children of the node are the content of the tab. The tab that is active is the only one visible. 
```

Comment:
```suggestion
		The tabs are generated only by the first children of the TabContainer node. Children of the node are the content of the tab. Only the content of the active tab is visible.
```

Reply:
I think this is basically the same as the first paragraph :P

> Arranges child controls into a tabbed view, creating a tab for each one. The active tab's corresponding control is made visible, while all other child controls are hidden. Ignores non-control children.